AI/finder/ticker_selector.py

```
@@ -0,0 +1,185 @@
+    '''
+    print("--- Top 3 종목 선정 시작 ---")
+
+    # 1. 뉴스 요약 데이터 파싱 및 종목별 집계
+    df = news_summary_df.copy()
+    df['parsed'] = df['summary'].apply(_parse_summary)
+
+    result = defaultdict(lambda: defaultdict(list))
+    for item in df['parsed'].dropna():
+        try:
+            stock = item.get("Stock").upper()
+        except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)